button.h

```diff
@@ -0,0 +1,19 @@
+#ifndef button_H_
+#define button_H_
+
+/********************************************************
+												includes
+***********************************************************/
+#include "services.h"
+
+#include "port.h"
+
+
+/********************************************************
+												prototypes
+***********************************************************/
+
+uint8_t BUTTON_READ	(enu_port_pin_num_t enu_port_pin_num, enu_port_num_t enu_port_num);
```

Comment:
Arguments in function prototype should start with "a" (indicator as an argument).